lib/rollbar.rb

```diff
@@ -619,6 +605,33 @@ def send_failsafe(message, exception)
       failsafe_payload
     end
 
+    def failsafe_body(message, exception)
+      body = 'Failsafe from rollbar-gem. '
+      log_error "[Rollbar] Sending failsafe response due to #{message}."
+
+      if exception
+        begin
+          backtrace = exception.backtrace || []
+          nearest_frame = backtrace[0]
+
+          exception_info = exception.class.name
+          exception_info += " in #{nearest_frame}" if nearest_frame
+
+          body += "#{exception_info}: #{message}"
+
+          log_error "[Rollbar] #{body}"
+        rescue
+        end
+      else
+        begin
+          body += message.to_s
+        rescue
+        end
+      end
```

Comment:
Couple things to avoid a potential 413 or 422 when the request is made to our API:

- Should truncate the resulting body in case it's too long
- Any chance of encoding issues?